rero_ils/modules/documents/dojson/contrib/marc21tojson/rero/model.py

```diff
@@ -589,18 +561,16 @@ def perform_subdivisions(field):
         if tag_key == '655':
             # remove the square brackets
             string_build = re.sub(r'^\[(.*)\]$', r'\1', string_build)
-        subject[field_data_per_tag[tag_key]] = string_build
+        subject['authorized_access_point'] = string_build
 
-        if tag_key in ['610', '611']:
-            subject['conference'] = conference_per_tag[tag_key]
-        elif tag_key in ['600t', '610t', '611t']:
+        if tag_key in ['600t', '610t', '611t']:
             creator_tag_key = tag_key[:3]  # to keep only tag:  600, 610, 611
-            subject['creator'] = remove_trailing_punctuation(
+            subject['authorized_access_point'] = remove_trailing_punctuation(
                 build_string_from_subfields(
                     value,
                     subfield_code_per_tag[creator_tag_key]),
                 '.', '.'
-            )
+            ) + '. ' + subject['authorized_access_point']
```

Comment:
Don't understand why we have to add `subject['authorized_access_point']` again.
If we really have to add it it is preferable to work with `f` strings.

Reply:
@rerowep in this case (600t, 610t, 611t), it seems some subfields value must be prepend to subject term.

Reply:
> If we really have to add it it is preferable to work with `f` strings.

I do not see a simple way to do this. Can you provide an example?